cmd/next.go

```diff
@@ -61,6 +62,10 @@ func nextPipeline(afs afero.Afero, writer io.Writer, part string, prerelease, me
 		return err
 	}
 
+	if part != "patch" && part != "minor" && part != "major" {
```

Comment:
I actually think we should create a func for the command.Args above to validate we get 1 parameter and it is part of the valid options. https://github.com/spf13/cobra/blob/5b2b9e9f61d36ccb66167301f76a2292c3729855/args.go#L8
Up on line 25, https://github.com/miniscruff/changie/pull/287/files#diff-c2a7c617a6418a3f91c597ddc318504bf0ec9fec0d4314f78ab67b34e14f3230R25

Reply:
ok done. Moved test case to e2e.

Reply:
Oh nice I actually didn't see that option, that is better than I expected